README.md

```diff
@@ -83,15 +83,15 @@ There aren't many published sharable components to drop in quite yet, as it's st
 
 ### Top-level API
 
-#### `components.newElement()`
+#### `components.HTMLElement`
 
 Creates a returns a new custom HTML element prototype, extending the HTMLElement prototype.
 
 Note that this does *not* register the element. To do that, call `components.registerElement` with an element name, and options (typically including the prototype returned here as your 'prototype' value).
 
 This is broadly equivalent to `Object.create(HTMLElement.prototype)` in browser land, and exactly equivalent here to `Object.create(components.dom.HTMLElement.prototype)`. You can call that yourself instead if you like, but it's a bit of a mouthful.
 
-#### `components.registerElement(componentName, options)`
+#### `components.customElements.define(componentName, Constructor)`
```

Comment:
Although it's simpler, I argue it's not cleaner. The `customElements` object would include extra, unrelated methods, the most notable of which being `HTMLElement`. This will matter when isometric elements come into the picture.

I pushed the update to a different branch; see [the readme](https://github.com/Concatapult/server-components/tree/elems-v1-renamed#define-a-component) to view the difference.

Another benefit is being able to destructure imports. Example:

```js
var { customElements, HTMLElement } = require('server-components');
// OR
import { customElements, HTMLElement } from 'server-components';
```